test/e2e/new-dns-record.spec.ts

```diff
@@ -16,27 +20,93 @@ test.describe('authenticated as user', () => {
     await page.goto('/dns-records/new');
   });
 
-  test('redirects to edit DNS record page when required fields are filled', async ({ page }) => {
-    await page.getByLabel('Name*').fill('test');
-    await page.getByRole('combobox', { name: 'Type' }).selectOption('A');
-    await page.getByLabel('Value*').fill('test');
+  /**
+   * Clean up after all tests are run
+   * For some weird reason using afterEach to delete the one record doesn't work
+   * The test is run before the records are deleted
+   * Thus, we are using different values for the fields for the tests
+   */
+  test.afterAll(async () => {
+    await prisma.dnsRecord.deleteMany();
+  });
+
+  interface DnsRecordData {
```

Comment:
Why not use the type from Prisma?  If you can't use it as is, at least consider using it's internal types (e.g., `DnsRecord['value']` vs. `string` etc)

Reply:
I actually forgot why I did that. I will try changing it to use the Prisma type

Reply:
I have resolved this and done a bunch of refactoring that also removed the need to have unique names for different records (https://github.com/DevelopingSpace/starchart/pull/576#discussion_r1165855497)